include/circt/Dialect/HW/HWOps.h

```diff
@@ -25,6 +25,8 @@
 #include "mlir/Interfaces/SideEffectInterfaces.h"
 #include "llvm/ADT/StringExtras.h"
 
+#include <map>
```

Comment:
Does `llvm::StringMap` not work for some reason? (If you haven't seen it, I use the [LLVM Programmer's Manual(https://www.llvm.org/docs/ProgrammersManual.html) for a guide to LLVM datastructures.)